apps/member-profile/app/shared/constants.server.ts

```diff
@@ -27,6 +29,8 @@ const BaseEnvironmentConfig = z.object({
 const EnvironmentConfig = z.discriminatedUnion('ENVIRONMENT', [
   BaseEnvironmentConfig.partial({
     CRUNCHBASE_BASIC_API_KEY: true,
+    GITHUB_OAUTH_CLIENT_ID: true,
+    GITHUB_OAUTH_CLIENT_SECRET: true,
```

Comment:
In the development environment, it's not required to have these environment variables set (b/c new contributors won't have them).